docs/1-qms/0-quality-manual-policy-objectives.md

```diff
@@ -0,0 +1,191 @@
+| IEC 62304:2006                                       | Software development for medical devices                           |
+| IEC 62366-1:2015                                     | Usability evaluation for medical devices                           |
+
+### Exclusions
+
+The following sections of ISO 13485:2016 will be excluded due to the product
+being stand-alone software:
+
+- 6.4.2 Contamination control
+- 7.5.2 Cleanliness of product
+- 7.5.5 Particular requirements for sterile medical devices
+- 7.5.7 Particular requirements for validation of processes for sterilization
+  and sterile barrier systems
+- 7.5.9.2 Particular requirements for implantable medical devices
+
+## 2. Quality Policy & Objectives
+
+### Quality Policy
+
+<!-- > Describe what your company is about, specifically, its mission and things
+> which are important for it. Maybe you're developing software for patients
+> with a certain disease and your goal is to improve their lives. -->
+
+{{device_manufacturer}} aims to make AI assisted radiotherapy cancer treatments
+accessible to all. With the goals to reduce treatment errors while improving
+overall treatment efficacy and efficiency.
+
+### Quality Objectives
+
+<!-- > Whatever policy you outlined above, now you need to make it measurable by
+> defining objectives which can be tracked. Those objectives should not (only)
+> refer to the quality of your devices but the quality of your QMS and the
+> overall work of your organization. > Typical examples are: hiring excellence
+> in staff, providing best-of-class device performance, high standards of
+> customer satisfaction, etc. In a next step (see short-term goals), those are
+> narrowed down to concrete measures like for example the monthly number of
+> user complaints. -->
+
+When tracking the quality of the AI autocontouring software its
+results are compared to health practitioner ground truth contours. These
+comparisons are undergone utilising the following metrics:
+
+- Hausdorff
+- Surface and Volumetric Dice
+- Four point health practitioner approval scale
+
+### Short-Term Goals
+
+<!-- > How does your team track its goals? Your auditors want to see how your
+> quality objectives translate into your daily work. You should formulate
+> strategic goals for your company that are somewhat related to your quality
+> goals and which are tracked at least on an annual basis. Do you already have
+> a goal-oriented system in place to track your team's work? Even better: align
+> business and quality goals and describe your system here. -->
+
+## 3. Roles
+
+<!-- > Describe the roles of the people in your company. Typically this is done by
+> drawing an organigram (you could use draw.io for that). Or, you just use a
+> table like below. -->
+
+| Role(s)                                     | People           |
+| ------------------------------------------- | ---------------- |
+| CEO, QMO, PRRC, CTO, CPO, Software Engineer | Simon Biggs      |
```

Comment:
Is there a reason why you aren't putting each role on a separate line and listing your name for each one.
At some point, and certainly as a copyable example, one would expect each role to be filled by a different person, or at least not all of them filled by one person.

Reply:
Good call. I just wasn't sure how best to do this one. I'll change them onto new lines as I step through the documents.